multiple-instances.md

```diff
@@ -3,17 +3,223 @@ It is possible to run multiple instances of AIO on one server.
+   </details>
+4. Configure your new VM:
+
+   After it has finished installing, the VM will have rebooted and presented you with a login prompt. For Debian, just use `root` as the username, and enter the password you chose during the installation process. Ubuntu restricts root account access, so you'll need to first login with your non-root user, and then run `sudo su -` to elevate your privileges.
+
+   We will now run a few commands to install docker and AIO in reverse proxy mode! As with any other commands, carefully read and try your best to understand them before running them.
+
+   **Each time you reach this step and run the `docker run` command below, you'll need to increment the `TALK_PORT` value. For example: 3478, 3479, etc... You may use other values as long as they don't conflict, and make sure they are [greater than 1024](https://github.com/nextcloud/all-in-one/discussions/2517). Be sure to note down the Talk port number you've assigned to this VM/AIO instance. You will need it later if you decide to enable Nextcloud Talk.**
+
+   Run these commands (**on the VM**):
+   ```shell
+   apt install -y curl
+   
+   curl -fsSL https://get.docker.com | sh
+
+   # Make sure you increment the TALK_PORT value every time you run this!
+   docker run \
+   --init \
+   --sig-proxy=false \
+   --name nextcloud-aio-mastercontainer \
+   --restart always \
+   --publish 8080:8080 \
+   --env APACHE_PORT=11000 \
+   --env APACHE_IP_BINDING=0.0.0.0 \
+   --env TALK_PORT=3478 \
+   --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
+   --volume /var/run/docker.sock:/var/run/docker.sock:ro \
+   nextcloud/all-in-one:latest
+   ```
+   The last command may take a few minutes. When it's finished, you should see a success message, saying "Initial startup of Nextcloud All-in-One complete!". Now exit the console session with `Ctrl + ]`. This concludes the setup for this particular VM.

```

Comment:
```suggestion
   The last command may take a few minutes. When it's finished, you should see a success message, saying "Initial startup of Nextcloud All-in-One complete!". Now exit the console session with `Ctrl + [c]`. This concludes the setup for this particular VM.

```

Reply:
Hi, sorry I didn't see this until now- but that wasn't a typo, the escape sequence really is `ctrl + closing square bracket`. When you use the `virsh console` command to connect to an existing VM, it prints this helpful message:
```shell
# virsh console --domain my-virtual-machine
Connected to domain 'my-virtual-machine'
Escape character is ^] (Ctrl + ])
```
But when you first create the VM using the `virt-install` command, you never get to see this message telling you how to exit the console. So that's why I felt it's worth mentioning.

Also, it's kinda buried but there's a brief mention of this in the [virsh manpage](https://gitlab.com/libvirt/libvirt/-/blob/master/docs/manpages/virsh.rst?plain=1#L89), where they at least mention why they chose such a weird sequence :)

I'm still hoping to submit another PR in a month or so when I have more free time- to include proper config for getting Talk working with the VMs (and hopefully auto borgbackups via the AIO interface too). So if you want I can just sneak this in along with that PR.

Reply:
> I'm still hoping to submit another PR in a month or so when I have more free time- to include proper config for getting Talk working with the VMs (and hopefully auto borgbackups via the AIO interface too). So if you want I can just sneak this in along with that PR.

Yes, would be great. Thanks! :)